pkg/transparentproxy/kubernetes/kubernetes.go

```diff
@@ -87,6 +87,7 @@ func NewPodRedirectForPod(pod *kube_core.Pod) (*PodRedirect, error) {
 	}
 
 	podRedirect.ExcludeInboundPorts, _ = metadata.Annotations(pod.Annotations).GetString(metadata.KumaTrafficExcludeInboundPorts)
```

Comment:
nit: maybe we can merge these 2 lines in something like a method `getExcludedInboundPorts` and just one assigning

Reply:
Yeah, `excludeVirtualProbePort` is that function